frontend/src/pages/warehouse/ModalAddWarehouse.js

```diff
@@ -127,63 +127,67 @@ function ModalAddWarehouse(props) {
 
     const addWarehouseHandler = (e) => {
```

Comment:
too big and nested function. Please, divide it into smaller ones.

Reply:
got it, thanks!